shared-bindings/supervisor/Runtime.c

```diff
@@ -80,8 +80,28 @@ const mp_obj_property_t supervisor_serial_connected_obj = {
               (mp_obj_t)&mp_const_none_obj},
 };
 
+/*Added to allow for polling of USB Console*/
```

Comment:
Please add a comment like the one above to document this new attribute. It starts with `//|     .. attribute:: runtime.serial_connected`